src/main/scala/com/fulcrumgenomics/cmdline/FgBioTool.scala

```diff
@@ -26,9 +26,51 @@ package com.fulcrumgenomics.cmdline
 import com.fulcrumgenomics.cmdline.FgBioMain.FailureException
 import com.fulcrumgenomics.commons.util.LazyLogging
 import com.fulcrumgenomics.sopt.cmdline.ValidationException
+import com.fulcrumgenomics.util.Metric
+import htsjdk.samtools.{SAMFileHeader, SAMProgramRecord}
+
+import scala.annotation.tailrec
+
+/** Stores meta information about the command line use to invoke a tool.
+  *
+  * @param name the name of the tool.
+  * @param args the list of arguments as given on the command line; this will contain any arguments also given to
+  *             [[FgBioMain]] in [[FgBioCommonArgs]].
+  * @param commandLineWithDefaults the command line as given to the tool, along with the defaults for all other
+  *              arguments.  This will include arguments given to [[FgBioMain]] in [[FgBioCommonArgs]].
```

Comment:
The definitions between these two seem very similar.

I think the `args` _**does not**_ have any arguments also given to `FgBioMain` in `FgBioCommonArgs`?